apps/alert_processor/lib/rules_engine/informed_entity_filter.ex

```diff
@@ -141,7 +141,7 @@ defmodule AlertProcessor.InformedEntityFilter do
   end
 
   defp trip_departure(subscription, schedule) do
-    case Enum.find(schedule, &(&1.stop_id == subscription.origin)) do
+    case Enum.find(schedule, &(&1.stop_id == subscription.origin && &1.departure_time)) do
```

Comment:
tiny suggestion: To be more explicit about the type of the value, we could write this condition as

```elixir
&1.stop_id == subscription.origin and not is_nil(&1.departure_time)
```